valid-palindrome/naringst.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @param {string} s
+ * @return {boolean}
+ */
+
+/**
+ * Runtime: 66ms, Memory: 54.75MB
+ * Time complexity: O(s.length)
+ * Space complexity: O(s.length)
+ *
+ */
+
+var isPalindrome = function (s) {
+  let trimmed = s.toLowerCase();
+  let answer = [];
+  let checkAlphabet = /[a-zA-Z]/;
+  let checkNum = /[0-9]/;
```

Comment:
```suggestion
  const trimmed = s.toLowerCase();
  const answer = [];
  const checkAlphabet = /[a-zA-Z]/;
  const checkNum = /[0-9]/;
```

재할당되지 않은 변수를 const 키워드로 선언하는 것을 추천합니다!
가독성과 이후 의도지 않은 버그 발생 여지 사단 차단한다는 점에서 장점을 가지고 있어요!

Reply:
이전 로직에서 재할당하려고 let으로 작성했는데 바꾸는걸 까먹었군요 ㅎㅎ 감사합니다!